nav2_system_tests/maps/warehouse_keepout_mask.yaml

```diff
@@ -0,0 +1,6 @@
+image: warehouse_keepout_mask.png
```

Comment:
You can remove any of the turtlebot sandbox world files in this directory too now!

Reply:
Basically anywhere in bringup/system tests that have the TB3 world stuff, remove it with the replaced files

Reply:
Perfect! I was waiting for your authorization 😄